app/Models/Facility.php

```diff
@@ -43,17 +44,14 @@ class Facility extends Model
     protected static function fieldValidator(string $field): string|array
     {
         return match ($field) {
-            'name' => 'required|string',
-            'url' => [
-                'bail',
-                'required',
-                'string',
-                'max:15',
+            'name' => Valid::trimmed(),
+            'url' => Valid::trimmed([
+                'max:30',
```

Comment:
Teraz zauważyłem, że w tabeli jest 36, a tutaj 30. Czy to było zamierzone?

Reply:
Byle nie było w przeciwną stronę :p

Wolę jakoś dać parę znaków zapasu